tests/BuildScriptGenerator.Tests/Ruby/RubyPlatformTests.cs

```diff
@@ -198,6 +198,42 @@ public void GeneratedBuildSnippet_DoesNotContainsJekyllCommand_WhenOnlyGemFileEx
             Assert.Contains("bundle install", buildScriptSnippet.BashBuildScriptSnippet);
         }
 
+        [Fact]
+        public void GeneratedBuildSnippet_CustomBuildCommandWillExecute_WhenOtherCommandsAlsoExist()
```

Comment:
can you add some buildimage tests for vso-focal and jamstack image to make sure we are covering the end to end scenarios here?